app/code/local/Magmodules/Webwinkelconnect/Model/Observer.php

```diff
@@ -158,5 +158,30 @@ public function processInvitationcall($observer)
             }
         }
     }
+    public function sendSyncUrl(): void {
+        $helper = Mage::helper('webwinkelconnect');
+        if (!$helper->isProductReviewInviteEnabled()) {
+            return;
+        }
+
+        $curl = new Varien_Http_Adapter_Curl();
+        $curl->setConfig([
+            'timeout'   => 10,
+        ]);
+        $url = 'https://dashboard.webwinkelkeur.nl/webshops/sync_url';
+        $data = json_encode([
+            'webshop_id' => $helper->getShopId(),
+            'api_key' => $helper->getApiKey(),
+            'url' => Mage::getBaseUrl(Mage_Core_Model_Store::URL_TYPE_WEB) . 'webwinkelkeur/index/sync',
+        ]);
+
+        $curl->write(Zend_Http_Client::POST, $url, '2', ['Content-Type:application/json'], $data);
+        $response = $curl->read();
+        if (!$response) {
+            $curl->close();
+            throw new Exception(sprintf('Could not send sync URL to dashboard: (%s) %s',$curl->getErrno(), $curl->getError()));
```

Comment:
Is throwing an exception the right thing to do here? What does that do to the user experience in this case, when they save the config and submitting the URL fails?

Reply:
I think throwing an exception is the right thing to do here. What happens is the settings that are changed in the config are saved, but an error notification is shown to the user that the sync URL is not sent to the dashboard and an exception log is generated for debugging. 
![image](https://github.com/webwinkelkeur/magento1/assets/44569402/8322f887-f4f0-4d48-afc2-bee6d4215926)

Reply:
Not sure what is the common way to show errors in magento1... but it looks good to me in this example. 
Hopefully, this is not shown only in some kind of debug mode?